server/matchmaker/matchmaker_queue.py

```diff
@@ -49,6 +49,7 @@ def __init__(
         featured_mod: str,
         rating_type: str,
         team_size: int = 1,
+        params: Optional[Dict[str, Any]] = None,
```

Comment:
Does it really make sense to use Any here? Ultimately these will all become strings for the command line anyway.

Reply:
The type annotations are just for internal typechecking. They should reflect the type of the variable accurately, although I'm just a bit too lazy to do `Union[str, int, bool]` here so I just do `Any`. All this typing stuff will get a bit nicer in python 3.9

Reply:
I actually meant why do you want to have any other types than str anyway?

Reply:
I guess because I still expect the params object to contain any valid json value